axelor-base/src/main/resources/views/CurrencyConversionLine.xml

```diff
@@ -17,7 +17,7 @@
         <panel name="main">
 		  <field name="startCurrency" widget="SuggestBox" form-view="currency-form" grid-view="currency-grid"/>
 		  <field name="endCurrency" widget="SuggestBox" form-view="currency-form" grid-view="currency-grid"/>
-		  <button name="getRate" title="Get rate" readonlyIf="startCurrency == null || endCurrency == null || toDate != null || id == null"   onClick="action-group-base-currencyconversionline-getrate-click"/>
+		  <button name="getRate" title="Get rate" readonlyIf="startCurrency == null || endCurrency == null || toDate != null"   onClick="action-group-base-currencyconversionline-getrate-click"/>
 		</panel> 
```

Comment:
Previous line rate is also checked for variation calculation. So changes are required on  [CurrencyConversionLineController.java](https://github.com/axelor/axelor-business-suite/blob/dev/axelor-base/src/main/java/com/axelor/apps/base/web/CurrencyConversionLineController.java#L68) too.

Reply:
Not if the id is null: https://github.com/axelor/axelor-business-suite/blob/dev/axelor-base/src/main/java/com/axelor/apps/base/web/CurrencyConversionLineController.java#L60